armi/nuclearDataIO/cccc/isotxs.py

```diff
@@ -123,7 +124,10 @@ def compare(lib1, lib2, tolerance=0.0, verbose=False):
                 runLog.warning(warning.format(nuc2, 2, 1))
             equal = False
             continue
-        equal &= compareNuclideXS(nuc1, nuc2, tolerance, verbose)
+        nucEqual = compareNuclideXS(nuc1, nuc2, tolerance, verbose)
```

Comment:
Could consider expanding in this compareNuclideXS function to write out all differences or plot them for further assessments.

Reply:
verbose prints the matricies. there are often hundreds of differences and hundreds of plots im not sure are that benifical.